toolchain/cc_toolchain_config.bzl

```diff
@@ -146,6 +146,7 @@ def _impl(ctx):
                         flags = [
                             bit_flag,
                             "-Wall",
+                            "-Wundef",
```

Comment:
I don't think this does what you think it does.  These flags only apply during our release process, and only affect our precompiled protoc artifacts

Reply:
Is there another place where flags are applied such that they are checked in CI/CQ? It would be great to have some enforcement so that the warning doesn't pop back up in the future.

Reply:
You should add it here: https://github.com/protocolbuffers/protobuf/blob/main/ci/common.bazelrc to apply it to CI

Reply:
This isn't resolved, this won't actually do anything